### ISSUE TITLE
@W-14619658 [Accessibility] Screen reader issues with hover and on focus content, Superbadges & Super Sets - View Superbadge, Prerequisite

### ISSUE THREAD

**Issue by mlopez-sf** [salesforce/design-system-react#3174]
When tabbing to the prerequisite links there is a tooltip. The following accessibility issues were observed:

Pointer hover can trigger the additional content, but the pointer can’t be moved over the additional content without the additional content disappearing. Keyboard users are forced to navigate back to the trigger element or out of the additional content in order to dismiss it.

- JAWS announces the content “Complete this badge to unlock superbadge” before the link text and then re-announces the hover content again.
- NVDA does not announce the content “Complete this badge to unlock superbadge” at all.  The content can't be dismissed without moving the pointer or keyboard focus.
